scripts/travis/deploy_packages.sh

```diff
@@ -12,4 +12,6 @@ set -e
 scripts/travis/build.sh
 
 export RELEASE_GENESIS_PROCESS=true
+export NO_BUILD=true
+export SkipCleanCheck=1
```

Comment:
assumption of clean build on travis?

Reply:
No. the `scripts/travis/build.sh` before already perform a complete build, plus check the enlistment.
so I was avoiding doing the same work twice. ( which is meaningful on slow arm builds )